entrypoint/createcluster/createcluster_suite_test.go

```diff
@@ -20,7 +22,12 @@ func TestMain(m *testing.M) {
 	flag.Var(&customflag.ServiceFlag.ClusterConfig.Destroy, "destroy", "Destroy cluster after test")
 	flag.Parse()
 
-	cfg, err = config.AddConfigEnv("../../config")
+	configPath, err := shared.EnvDir("entrypoint")
+	if err != nil {
+		return
+	}
+	fmt.Println("configPath: ", configPath)
```

Comment:
https://github.com/rancher/distros-test-framework/pull/32/files#r1352755686

Reply:
this should not be here
tks